snuba/query/logical.py

```diff
@@ -83,25 +77,15 @@ def __init__(
         )
 
     def _get_expressions_impl(self) -> Iterable[Expression]:
```

Comment:
Can these be removed entirely?

Reply:
They are abstract in the parent class. I could provide a default implementation there, which I am not a big fan of because it makes it less clear why they exist to begin with. I would not in this case because we are likely to have at most 3-4 classes for queries. So this is not going to clutter the implementations.

Reply:
Got it. OK cool.